src/lightning/pytorch/strategies/launchers/xla.py

```diff
@@ -68,8 +68,7 @@ def launch(self, function: Callable, *args: Any, trainer: Optional["pl.Trainer"]
                 a selected set of attributes get restored in the main process after processes join.
             **kwargs: Optional keyword arguments to be passed to the given function.
         """
-        context = mp.get_context(self._start_method)
-        return_queue = context.SimpleQueue()
```

Comment:
The `return_queue` annotation for `_wrapping_function` needs an update. I don't thin k `SimpleQueue` and `Queue` have a shared interface